sa/storage-authority.go

```diff
@@ -49,22 +50,39 @@ type authzModel struct {
 }
 
 // NewSQLStorageAuthority provides persistence using a SQL backend for Boulder.
-func NewSQLStorageAuthority(driver string, dbConnect string) (ssa *SQLStorageAuthority, err error) {
+func NewSQLStorageAuthority(driver string, dbConnect string) (*SQLStorageAuthority, error) {
 	logger := blog.GetAuditLogger()
 	logger.Notice("Storage Authority Starting")
 
+	if driver == "mysql" {
+		// Check the parseTime=true DSN is present
+		dbURI, err := url.Parse(dbConnect)
+		if err != nil {
+			return nil, err
+		}
+		dsnVals, err := url.ParseQuery(dbURI.RawQuery)
```

Comment:
I feel like this may be overkill. Why not just error out if driver == "mysql" and "parseTime=true" isn't in the dbConnect?